pkg/render/common/elasticsearch/service.go

```diff
@@ -25,13 +25,30 @@ const (
 	linseedEndpoint     = "https://tigera-linseed.%s.svc"
 	httpsFQDNEndpoint   = "https://tigera-secure-es-gateway-http.%s.svc.%s:9200"
 	linseedFQDNEndpoint = "https://tigera-linseed.%s.svc.%s"
+
+	// Handle for multi tenant
+	guardianEndpoint       = "https://guardian.calico-system.svc"
```

Comment:
nit: all the constants that are declared are used exactly one time. The result is that the switch statement is actually very hard to read; if they didn't exist at all, I can read the code and see what the urls look like as I am reading the code.